### ISSUE TITLE
[1.10.2]dupe like bug with tinker-io

### ISSUE THREAD

**Issue by Tsukiumy** [xLionel775/AgriculturalExpansion#17]
whenever you put Agricultural expansion toghether with the tinker-io addon, all modded ingots are doubled, as stated here
https://github.com/gkbm2013/tinker-IO/issues/24
TConstruct-1.10.2-2.4.0
Tinker I/O beta 2.3.5
AgriculturalExpansion[1.9.4][1.10.2]-r1.2.1
 the bug:
for some odd reason the smart output is doubling ores when it shoudn't like it
http://imgur.com/a/D1rEs
some images to explain the bug, vanilla ores are OK, but the modded no.

**Comment by ionelaipatioaei** [comment]
Yeah it's a known bug. It is only visual, I don't think it affects the output. 
I'll fix it soon but I have no time right now, I'll leave the issue open until then.

**Comment by Tsukiumy** [comment]
in real it really afects, its some weird interaction, cause until it leaves the internal tank, it still with the correct quantity, but at the casting process(its automatic inside the refered tank) it doubles the ore, not a visual bug.
